.github/workflows/e2e_test_ci_models.yml

```diff
@@ -35,7 +35,7 @@ jobs:
             target_os: "linux"
             target_arch: "x86_64"
           - name: "macOS aarch64 (Apple Silicon)"
-            builder: "macos-14"
+            builder: [self-hosted, macOS]
```

Comment:
Can we rename this from `self-hosted` to `spice-hosted`?

Reply:
Updated to use existing label `spiceai-macos` (created yesterday after PR)